js/models/order/Case.js

```diff
@@ -85,7 +85,7 @@ export default class extends BaseOrder {
         const coinDivisibility = listing.metadata
           .coinDivisibility;
 
-        item.quantity = item.quantity / coinDivisibility;
+        item.quantity = item.quantity64 / coinDivisibility;
```

Comment:
I think this should be fixed in parse, similar to how it's done in the Order model.

```
        // standardize the quantity field
        item.quantity = item.quantity === 0 ?
          item.quantity64 : item.quantity;
```

If I remember correctly, old nodes would return quantity whereas new ones would return the quantity64. Probably shouldn't be many or any nodes that old, but it doesn't hurt to account for both.

Reply:
It's in the convertCryptoQuantity method, which is being called by parse. 

I originally had the code above, but since it's only being called when the contractType is cryptocurrency, it felt like it wasn't necessary.

Reply:
I see what you're saying now. I was thinking the quantity64 was just for crypto listings, I forgot it's for all newer listings.